Author: Fabian Greffrath <fabian+debian@greffrath.com>
Description: Avoid undefined behaviour when strcpy(3) is
 called with the same pointer as source and destination.
 Thanks, Jakub Wilk.
Bug-Debian: https://bugs.debian.org/775132

--- a/source/base/all/lfn/lfn.c
+++ b/source/base/all/lfn/lfn.c
@@ -30,8 +30,6 @@ INT     BASE_LFN_ChangeDir(PCHAR Directo
 PCHAR   BASE_LFN_GetCurrentDir(PCHAR Directory, size_t Size)
 {
   Directory = (PCHAR) getcwd(Directory, Size);
-  
-  strcpy(Directory, BASE_LFN_LongName(Directory));
 
   return Directory;
 }
